ordering/stories/ProductList.js

```diff
@@ -0,0 +1,68 @@
+import { storiesOf } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+import { reduxForm, Field } from 'redux-form'
+import h from 'react-hyperscript'
+
+import ProductList from '../components/ProductList'
+
+// https://github.com/root-systems/cobuy/wiki/Models
+const mockProductList = [
+  {
+    name: 'crayons',
+    description: 'these are crayons',
+    image: 'http://www.mercurius-australia.com/site/images/1250623.jpg',
+    priceSpecifications: [
```

Comment:
i changed this to `priceSpecs` (because it felt like soooooooo long to type), but happy to be convinced to change it back (since my other half of my self thinks sortened names are lazy). but we'll need to follow suit in the product editor code, which at the moment is using `priceSpec`.